Sources/AppStoreConnectCLI/Services/Operations/ListBetaTestersByGroupOperation.swift

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+
+struct ListBetaTestersByGroupOperation: APIOperation {
+
+    struct Options {
+        let groupId: String
+    }
+
+    private let options: Options
+
+    typealias BetaTester =  AppStoreConnect_Swift_SDK.BetaTester
+    typealias Output = [BetaTester]
+
+    init(options: Options) {
+        self.options = options
+    }
+
+    func execute(with requestor: EndpointRequestor) throws -> AnyPublisher<Output, Error> {
+        let endpoint = APIEndpoint.betaTesters(inBetaGroupWithId: options.groupId)
+
+        return requestor.request(endpoint)
```

Comment:
```suggestion
        return requestor.requestAllPages {
            .betaTesters(inBetaGroupWithId: options.groupId, next: $0)
        }
```
We have implemented the pagination logic so you might want to hook it up